Cargo.toml

```diff
@@ -10,12 +10,15 @@ categories = ["command-line-utilities", "text-processing"]
 edition = "2018"
 
 [dependencies]
+ahash = "0.3"
```

Comment:
Let's have these unrelated changes in a separate commit or PR.

Reply:
Done.